Code/Framework/AzToolsFramework/Platform/Linux/AzToolsFramework/API/PythonLoader_Linux.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/API/EditorPythonConsoleBus.h>
+#include <dlfcn.h>
+
+namespace AzToolsFramework::EmbeddedPython
+{
+    class PythonLoader
+    {
+    public:
+        PythonLoader()
+        {
+            m_embeddedLibPythonHandle = dlopen("libpython3.7m.so.1.0", RTLD_NOW | RTLD_GLOBAL);
+            if (!m_embeddedLibPythonHandle)
+            {
+                char* err = dlerror();
+                if (err)
+                {
+                    AZ_Error("PythonLoader", false, "Failed to load 'libpython' with error: %s\n", err);
+                }
+            }
```

Comment:
Instead of NOT triggering an error if dlerror() is NULL, you should trigger an error anyways, because m_embeddedLibPythonHandle() being null is clearly an error

```suggestion
                char* err = dlerror();
                AZ_Error("PythonLoader", false, "Failed to load 'libpython' with error: %s\n", err?err:"Unknown Error");
            }
```

Reply:
+1 on @spham-amzn comment.

Reply:
Nice, will do.